app/assets/javascripts/admin/addon/components/emoji-value-list.js

```diff
@@ -0,0 +1,189 @@
+import Component from "@ember/component";
+import I18n from "I18n";
+import { on } from "discourse-common/utils/decorators";
+import { emojiUrlFor } from "discourse/lib/text";
+import { action, set } from "@ember/object";
+import { later, schedule } from "@ember/runloop";
+
+export default Component.extend({
+  classNameBindings: [":value-list"],
+  collection: null,
+  values: null,
+  validationMessage: null,
+  emojiPickerIsActive: false,
+  isEditorFocused: false,
+  emojiName: null,
+
+  init() {
+    this._super(...arguments);
+    this.set("collection", []);
+  },
+
+  @action
+  emojiSelected(code) {
+    this.set("emojiName", code);
+    this.set("emojiPickerIsActive", !this.emojiPickerIsActive);
+    this.set("isEditorFocused", !this.isEditorFocused);
+  },
+
+  @action
+  openEmojiPicker() {
+    this.set("isEditorFocused", !this.isEditorFocused);
+    later(() => {
+      this.set("emojiPickerIsActive", !this.emojiPickerIsActive);
+    }, 100);
+  },
+
+  @action
+  clearInput() {
+    this.set("emojiName", "");
+  },
+
+  @on("didReceiveAttrs")
+  _setupCollection() {
+    this.set("collection", this._splitValues(this.values));
+  },
+
+  _splitValues(values) {
+    if (values && values.length) {
+      const emojiList = [];
+      const emojis = values.split("|");
+      emojis.forEach((emojiName, index) => {
+        const emoji = {};
+        emoji.value = emojiName;
+        emoji.emojiUrl = emojiUrlFor(emojiName);
+        emoji.isEditable = true;
```

Comment:
just put this in object creation.

```
const emoji =  { isEditable: true, isEditing: false };
```

We want to put the focus on what's dynamic.